extensions/open-remote-ssh/src/serverSetup.ts

```diff
@@ -149,8 +149,13 @@ export async function installCodeServer(conn: SSHConnection, serverDownloadUrlTe
     }
 
     const exitCode = parseInt(resultMap.exitCode, 10);
-    if (exitCode !== 0) {
-        throw new ServerInstallError(`Couldn't install vscode server on remote server, install script returned non-zero exit status`);
+    if (exitCode === 66) {
+        throw new ServerInstallError(
+            `There is no available Positron server with version '${installOptions.version}' for ${resultMap.platform} (${resultMap.arch}).`
+            + `\nPlease check the [system requirements](https://positron.posit.co/remote-ssh.html#system-requirements) and [troubleshooting guides](https://positron.posit.co/remote-ssh.html#how-it-works-troubleshooting).`
```

Comment:
Are these links to docs sufficient?

Reply:
I think those are helpful links!

Can we refine this a smidge so we don't end up with nested parentheses in the popup? Maybe just this is better:

```
There is no available Positron server with version ${installOptions.version} for ${resultMap.platform} ${resultMap.arch}.
```